src/platform/plugins/shared/metrics_experience/server/routes/dimensions/route.ts

```diff
@@ -16,25 +16,16 @@ import { getDimensions } from './get_dimentions';
 import { throwNotFoundIfMetricsExperienceDisabled } from '../../lib/utils';
 
 export const getDimensionsRoute = createRoute({
-  endpoint: 'GET /internal/metrics_experience/dimensions',
+  endpoint: 'POST /internal/metrics_experience/dimensions',
```

Comment:
I'm a bit on the fence about turning this into a POST request, even though I understand this was done to allow us to pass the list of metrics